install_libs.sh

```diff
@@ -68,18 +69,16 @@ sh autogen.sh --prefix=${PREFIX} --host=${CROSS_TC}
 ./configure --prefix=${PREFIX} --host=${CROSS_TC}
 make -j$PARALLEL_JOBS && make install
 
-
 #zstd
 REPO=https://github.com/facebook/zstd
 LOCALREPO=zstd
 get_clean_repo
 
 mkdir -p ${LIBDIR}/libs/${LOCALREPO}/build/cmake/build
 cd ${LIBDIR}/libs/${LOCALREPO}/build/cmake/build
-cmake -D ADDITIONAL_CXX_FLAGS="-lrt" -D CMAKE_BUILD_TYPE=RELEASE -D CMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_TOOLCHAIN_FILE=${LIBDIR}/${CROSS_TC}.cmake -DENABLE_NEON=ON -DNEON_INTRINSICS=ON ..
```

Comment:
for some reasons.... without it I had no issues with my Kobo Glo, but the Libra h2o had some issue at runtime "error while loading shared libraries: /mnt/onboard/.adds/qt-linux-5.15-kde-kobo/lib/libzstd.so.1: unexpected reloc type 0x03" 
Which actually makes more sense to me than having the Kobo Glo runs fine without it.